add-graphics-renderer/build.gradle

```diff
@@ -20,5 +20,5 @@ android {
 }
 
 dependencies {
-    compile 'com.android.support:appcompat-v7:23.2.1'
+    compile 'com.android.support:appcompat-v7:23.3.0'
```

Comment:
Would be nice to use a consistent version - should we update all to the newer version?